deploy/manifests/99_openshift-machineconfig_99-additional-master-ssh.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  creationTimestamp: null
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-additional-master-ssh
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    passwd:
+      users:
+      - name: core
+        sshAuthorizedKeys:
+        - REPLACE_SSH_KEY
```

Comment:
Any reason to not use an actual [go template](https://golang.org/pkg/text/template/)?

Reply:
How you will process it? cluster isn't up yet (`oc process` won't work)

Reply:
At first I was updating a current manifest so it wasn't an option, later we decided to create a new manifest for it and now it is a good idea.

Updated to use template.